var/spack/repos/builtin/packages/onednn/package.py

```diff
@@ -16,6 +16,11 @@ class Onednn(CMakePackage):
 
     maintainers = ['adamjstewart']
 
+    version('master', branch='master')
```

Comment:
I think you need to add a `git` attribute to be able to specify a `master` version tied to a branch.

Reply:
Fixed